app/views/projects/new.html.slim

```diff
@@ -31,11 +31,12 @@ p.well Please make sure you have read our #{link_to "mentor guide", 'http://rail
       = f.input :source_url, as: :url, wrapper_html: { class: 'col-md-6' }, label: "Repository", placeholder: 'Link to your source code (Github, Gitlab, Bitbucket, …)' 
       = f.input :description, wrapper_html: { class: 'col-md-6' }, placeholder: "What's your project about?", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
       = f.input :issues_and_features, wrapper_html: { class: 'col-md-6' }, placeholder: "A list of possible features to work on, maybe even concrete Github issues", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
+      = f.input :requirements, wrapper_html: { class: 'col-md-6' }, placeholder: "E.g. good knowledge of a specific programming language, proficiency with CSS, etc. Maybe a coding challenge (on Exercism.io, Codeschool, Codewars, etc.) that needs to be completed.", input_html: { rows: 6 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
```

Comment:
Valid point about *"basic"* and *"ideal"* - maybe this should just be left to the mentors themselves to decide? It probably also depends on the project: for some, defining a minimum entry barrier may be crucial to guarantee students picking the project will actually be able to do something within the 3 months, other may be more flexible different skill levels.

As for the word itself: I really like *"requirements"* and think it's a choice that fits. Don't see a benefit in changing it to *"prerequisites"*. To me this sounds more like *"you should have your machine set up to work with this, you should have had a look at the Readme and CoC"*.